src/Http/Exception/ProxyNotAllowedException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Http\Exception;
+
+use Exception;
+use Flarum\Foundation\KnownError;
+
+class ProxyNotAllowedException extends Exception implements KnownError
```

Comment:
We might want to have a custom error page for this, so that it's clear why the forum can't be accessed (otherwise it'd be the standard "an error has occured" page)

Reply:
A normal user wouldn't ever get this error if the server configuration is correct, so I think no custom error page is needed. 
This is a just a security mechanism.

Reply:
That... is absolutely correct...